src/termwidget.cpp

```diff
@@ -296,10 +296,10 @@ TermWidget::TermWidget(TerminalConfig &cfg, QWidget * parent)
 
     propertiesChanged();
 
-    connect(m_term, SIGNAL(finished()), this, SIGNAL(finished()));
-    connect(m_term, SIGNAL(termGetFocus()), this, SLOT(term_termGetFocus()));
-    connect(m_term, SIGNAL(termLostFocus()), this, SLOT(term_termLostFocus()));
-    connect(m_term, &QTermWidget::titleChanged, this, [this] { emit termTitleChanged(m_term->title(), m_term->icon()); });
```

Comment:
Why removing this (and adding `TermWidget::term_termTitleChanged()`)?

Reply:
Yeah, that's not necessary.